lightningd/jsonrpc.h

```diff
@@ -127,7 +127,7 @@ static inline void fixme_ignore(const struct command_result *res)
 }
 
 /* FIXME: For the few cases where return value is indeterminate */
```

Comment:
is the FIXME still valid, if you've annotated them?

Reply:
It's still a bad idea, though.  Just because I don't know how to fix it right now doesn't make it less of a FIXME :)